src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/dailykuudra/DailyKuudraBossHelper.kt

```diff
@@ -68,10 +67,10 @@ class DailyKuudraBossHelper(private val reputationHelper: CrimsonIsleReputationH
     fun render(display: MutableList<List<Any>>) {
         val done = kuudraTiers.count { it.doneToday }
         display.addAsSingletonList("")
-        display.addAsSingletonList("§7Daily Kuudra (§e$done§8/§e3 killed§7)")
-        if (done != 2) {
+        display.addAsSingletonList("§7Daily Kuudra (§e$done§8/§e5 killed§7)")
+        if (done < 5) {
             for (tier in kuudraTiers) {
-                val result = if (tier.doneToday) "§7Done" else "§bTodo"
+                val result = if (tier.doneToday) "§aDone" else "§bTodo"
```

Comment:
The Reputation helper uses gray as completed/done instead of green everywhere

Reply:
It uses green in the miniboss helper aswell (line 83: https://github.com/hannibal002/SkyHanni/blob/beta/src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/miniboss/DailyMiniBossHelper.kt) (sorry it's my first time contributing so I don't know if there's a way to mention code from other files)

Reply:
Indeed. It looks like the color scheme is not perfect there.